data/_models/shared/Organisations.csv

```diff
@@ -1,12 +1,12 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Organisations,,,,,,,,,,,"org:Organization,foaf:Agent",Research departments and research groups. Null values will be generated from RoR,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
-Organisations,,resource,ref,1,TRUE,,Resources,,,,dcterms:isPartOf,Resource that this organisation is part of,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
+Organisations,,,,,,,,,,,foaf:Agent,Research departments and research groups. Null values will be generated from RoR,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
+Organisations,,resource,ref,1,TRUE,,Resources,,,,,Resource that this organisation is part of,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
 Organisations,,id,,1,TRUE,,,,,,dcterms:identifier,Internal identifier,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
 Organisations,,pid,hyperlink,,,,,,,,dcterms:identifier,"Persistent identifier, ideally from ROR","DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
-Organisations,,name,text,,TRUE,,,,,,"dcterms:title,foaf:name",Name used in (inter)national projects,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
-Organisations,,acronym,,,,,,,,,dcterms:identifier,"Acronym, if applicable","DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
+Organisations,,name,text,,TRUE,,,,,,foaf:name,Name used in (inter)national projects,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
+Organisations,,acronym,,,,,,,,,,"Acronym, if applicable","DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
 Organisations,,logo,file,,,,,,,,,Logo of the organisation,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",
-Organisations,,country,ontology_array,,,CatalogueOntologies,Countries,,,,vcard:country-name,Country in which the institution head office or coordinating centre is located,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
```

Comment:
ook hier is niets mis mee volgens mij?

Reply:
I am not sure, see this about locations in Organization class: https://www.w3.org/TR/vocab-org/#location_information. Should we annotate this attribute with org:hasSite and the Countries table with org:Site?

Reply:
ah, probably need sem experts opinion @esthervanenckevort ?

Reply:
The countries table has countries in it, which are geographical regions. A site in the context of an organisation is something different, a site has usually has a siteAddress.
Semantically you could model it as follows

`
<https://www.umcg.nl/> a org:Organization
   hasSite _addr
      siteAddress: [ 
       vcard:hasAddress [
           vcard:country-name: "NL" 
        ]
      ]
    ] .
`

You could also see the Organisation as an instance of a vCard (next to Organization) with vCard type "Organization" and use the hasAddress property directly on the RDF.

`
<https://www.umcg.nl/> a org:Organization, vcard:Organization
   vcard:hasAddress [
       vcard:country-name: "NL" 
   ] .
`

However in both cases the country cannot be a direct property of Organisation.

If you would like to have country at the level of Organisation you should define your own property that has a org:Organization as domain.

I inlined blank nodes for brevity.

Reply:
I am not sure what to do here still. Better not to include country in the rdf output if it is not asked for anywhere? Or do we want to have it at the level of Organization for other purposes, maybe for ingesting data? And how do we define our "own property that has a org:Organization as domain"?